ai/Gemini_diary_summary.py

```
@@ -0,0 +1,69 @@
+from google import genai
```

Pylint (3.3.7)

[error] No name 'genai' in module 'google'

(E0611)